README.md

```diff
@@ -96,6 +96,36 @@ UI.
 Conduit can be configured through command line parameters. To view the full list
 of available options, run `./conduit --help`.
 
+### Homebrew
+
+Make sure you have [homebrew](https://brew.sh/) installed on your machine, then run:
+
+```sh
+brew install conduit
+brew upgrade conduit
+```
+
+### Debian
+
+Download the right `.deb` file for your machine architecture from the
+[latest release](https://github.com/conduitio/conduit/releases/latest), then run:
+
+```sh
+apt-get install dpkg
+dpkg -i package_name.deb # replace with the downloaded package name
+apt-get update
+apt-get install conduit
```

Comment:
Do we need all the `apt-get` stuff? I didn't work with Linux for a while, but it seems that `dpkg -i` should be enough to install the package..?

Reply:
I remember having to update `apt` before installing conduit, but I'll try running Linux on docker and double check

Reply:
oh it works!

Reply:
I'm far from being a Linux expert:) but yeah `apt` shouldn't be needed. Maybe we don't need to mention `apt-get install dpkg`, most will have it installed already.